docs/src/pages/[platform]/components/switchfield/SwitchPropControls.tsx

```diff
@@ -58,7 +58,7 @@ export const SwitchPropControls: SwitchPropControlsInterface = ({
       <TextField
         placeholder="Set thumb color"
         name="thumbColor"
-        value={thumbColor}
+        value={thumbColor as string}
```

Comment:
Just a thought. What do you think if we define it with `interface SwitchPropControlsProps`? 

Something like this...

```ts
export interface SwitchPropControlsProps extends SwitchFieldProps {
  thumbColor: string;
  trackColor: string;
  trackCheckedColor: string;
  setThumbColor: (
    value: React.SetStateAction<SwitchFieldProps['thumbColor']>
  ) => void;
...
}
```

Reply:
+1, but to do that, I think you have to Omit thumbColor, trackColor and trackCheckedColor from SwitchFieldProps:

```ts
export interface SwitchPropControlsProps extends Omit<SwitchFieldProps, 'thumbColor | trackColor | trackCheckedColor> {
  thumbColor: string;
  trackColor: string;
  trackCheckedColor: string;
  setThumbColor: (
    value: React.SetStateAction<SwitchFieldProps['thumbColor']>
  ) => void;
...
}
```

Reply:
I've tried this but it breaks the switchfield demo component.